scripts/get-helm-3

```diff
@@ -19,8 +19,16 @@
 
 : ${BINARY_NAME:="helm"}
 : ${USE_SUDO:="true"}
+: ${DEBUG:="false"}
+: ${VERIFY_CHECKSUM:="true"}
+: ${VERIFY_SIGNATURES:="false"}
 : ${HELM_INSTALL_DIR:="/usr/local/bin"}
 
+HAS_CURL="$(type "curl" > /dev/null && echo true || echo false)"
+HAS_WGET="$(type "wget" > /dev/null && echo true || echo false)"
```

Comment:
I found this in my terminal while testing this script:

```
bash: line 28: type: wget: not found
```

If you change this to `&> /dev/null` for all `HAS_*` variables, this output will be silenced.

Reply:
You dont have wget ?? :)
Thanks, resolved